changelog/Microsoft.IdentityProtectionServices.json

```
@@ -1,5 +1,79 @@
 {
 	"changelog": [
+		{
+		      "ChangeList": [
+			{
+			  "Id": "68fb7e36-efee-4e07-a53c-06325d9e1e91",
+			  "ApiChange": "Enumeration",
+			  "ChangedApiName": "conditionalAccessExternalTenantsMembershipKind",
+			  "ChangeType": "Addition",
+			  "Description": "Added the [conditionalAccessExternalTenantsMembershipKind](https://docs.microsoft.com/en-us/?view=graph-rest-beta) resource",
```

This is a Graph studio issue, please replace with my suggestion. Thanks.
```suggestion
			  "Description": "Added the **conditionalAccessExternalTenantsMembershipKind** enumeration.",
```